test/integration/tlspolicy_controller_test.go

```diff
@@ -335,9 +372,62 @@ var _ = Describe("TLSPolicy", Ordered, func() {
 				cert2 := &certmanv1.Certificate{}
 				err = k8sClient.Get(ctx, client.ObjectKey{Name: "test2-tls-secret", Namespace: testNamespace}, cert2)
 				Expect(err).ToNot(HaveOccurred())
```

Comment:
Here, we should check the certs have the expected DNSNames:

cert1.Spec.DNsNames = ["test1.example.com"]
cert2.Spec.DNsNames = ["test2.example.com", "test2.example.com"]

Reply:
done.